app/validators/user_validator.rb

```diff
@@ -33,4 +36,17 @@ def valid_phone_number_contents(number, record)
 
     true
   end
+
+  def validate_presence(attribute, record)
+    if attribute.empty?
+      record.errors.add(attribute, " can't be blank")
+      return true
```

Comment:
Should it return `false` instead of `true`?

Reply:
It should return `false`. I'll fix it now! :+1: